build/root/.bazelrc

```diff
@@ -29,7 +29,8 @@ test:unit --features=race
 
 test:unit --build_tests_only
 test:unit --test_tag_filters=-e2e,-integration
-test:unit --flaky_test_attempts=3
+test:unit --cache_test_results=no
+test:unit --runs_per_test=5
```

Comment:
maybe we continue to allow caching, but we enforce this in the future?

Reply:
so anything that changes a test or what it is testing has to pass N times in a row in that job to merge? I like the theory. I wonder if things low in the dependency tree would blow out the test time.